nix/overlay.nix

```diff
@@ -1,19 +0,0 @@
-final: _:
```

Comment:
`nix/overlay.nix` is part of the API of this package.
We can't remove it.

Reply:
See example use here:
https://github.com/NorfairKing/smos/blob/8c7f97b3aba304e6d0ac6d36493f9c22600a4572/flake.nix#L131